src/main/java/frc/robot/subsystems/shooter/ShooterFlywheelIOSim.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.subsystems.shooter;
+
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+import frc.robot.Constants;
+
+public class ShooterFlywheelIOSim implements ShooterFlywheelIO
+{
```

Comment:
The voltage to drive the motor at maximum speed can be calculated. We'll set this up in the constructor, but we need some variables first:

```suggestion
{
    private final double UPPER_MOTOR_MAX_VOLTAGE;
    private final double LOWER_MOTOR_MAX_VOLTAGE;
```